### PR TITLE
data:SwapnilPatle

### PR DESCRIPTION
**Pull request by SwapnilPatle**
## Check List (Check all the applicable boxes) 

- [ ] My code follows the code style of this project.
- [ ] My change requires changes to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] All new and existing tests passed.
- [ ] This PR does not contain plagiarized content.
- [ ] The title of my pull request is a short description of the requested changes.

## Note to reviewers

<a href="https://gitpod.io/#https://github.com/EddieHubCommunity/LinkFree/pull/3494"></a>

### CODE REVIEW

**Comment by HarshDeep61034** [comment]
Looks Good @SwapnilPatle congrats on Your first PR and welcome to the open source World. 😄

**Review by Tejender1521 — APPROVED** [review]
Everything looks good

**Review by Aadarsh805 — CHANGES_REQUESTED** [review]
Please move this file to data directory, and also the name of the file must be same as your GitHub username, make it `SwapnilPatle.json`
If you change the file name to `data/SwapnilPatle.json` it'll also move it to the data directory as well

File: Swapnil Patle [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+    },
+    {
+      "name": "Follow me on Facebook",
+      "url": "https://www.facebook.com/mdswapnil.patle",
+      "icon": "FaFacebook"
+    },
+    {
+      "name": "Follow me on YouTube",
+      "url": "https://www.youtube.com/channel/UCn0F3wdlhYyS3G5mO8li8Jg",
+      "icon": "FaYouTube"
```
Comment (Aadarsh805) [inline Swapnil Patle:36]: fix the youtube icon, it should be `FaYoutube`

File: Swapnil Patle [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+"links": [
+    {
+      "name": "Follow me on Twitter",
+      "url": "https://twitter.com/SwapnilPatle01",
+      "icon": "FaTwitter"
+    },
+    {
+      "name": "Follow me on GitHub",
+      "url": "https://github.com/SwapnilPatle",
+      "icon": "FaGitHub"
```
Comment (Aadarsh805) [inline Swapnil Patle:16]: this icon should be `FaGithub`

**Review by Aadarsh805 — CHANGES_REQUESTED** [review]

**Pull request closed without merging**